.eslintrc.json

```diff
@@ -31,6 +45,11 @@
               "MethodDefinition": true,
               "ClassDeclaration": true
           }
+      }],
+      "jsx-a11y/anchor-is-valid": [ "error", {
+        "components": [ "Link" ],
+        "specialLink": [ "hrefLeft", "hrefRight", "to" ],
+        "aspects": [ "noHref", "invalidHref", "preferButton" ]
       }]
     }
   }
```

Comment:
I see you added a lot of new rules to your eslint file. Are these rules in sync with those your Learning Facilitator expects?